### ISSUE TITLE
Flake8 rule 'E999' reported an invalid point

### ISSUE THREAD

**Issue by bohdyone** [AtomLinter/linter-flake8#316]
Flake8 reported an invalid point for the rule `E999`, with the messge `SyntaxError: invalid syntax`.

Code example:
```
'''Find ICON DB's and build SQL for purging'''
import ConfigParser
import cx_Oracle
import os
from passwordManager import retrieve
import socket
```

Debug information:
Atom version: 1.12.7
Flake8 version: `3.0.4 (pyflakes: 1.2.3, mccabe: 0.5.2, pycodestyle: 2.0.0) CPython 3.5.2 on Darwin`

**Comment by Arcanemagus** [comment]
Please update your `flake8` version (`pip install -U flake8`). This particular error should be fixed in `flake8>=3.2.1`.

**Comment by Arcanemagus** [comment]
Closing due to a lack of response, if you are still having issues after updating `flake8` please provide code to reproduce this!
